templates/bridgeserver2.yaml

```diff
@@ -1043,6 +1048,35 @@ Resources:
                 - - "arn:aws:s3:::"
                   - !Ref AttachmentBucket
                   - "/*"
+  ParticipantFileBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+  ParticipantFileBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket:
+        Ref: ParticipantFileBucket
+      PolicyDocument:
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              AWS: "arn:aws:iam::325565585839:root"
+            Action:
+              - "s3:ListBucket"
+              - "s3:GetObject"
+              - "s3:PutObject"
+              - "s3:ListObjectsV2"
+              - "s3:DeleteObject"
+              - "s3:CreateMultipartUpload"
+            Resource:
+              - !Join
+                - ""
+                - - "arn:aws:s3:::"
+                  - !Ref ParticipantFileBucket
```

Comment:
Khai says this is necessary.

Reply:
I see that we are giving beanstalk instances full s3 bucket access therefore the bridge app should have access to this bucket via the [AWSIAMRole](https://github.com/Sage-Bionetworks/BridgeServer2-infra/blob/develop/templates/bridgeserver2.yaml#L540).  In that case this policy is not needed.

from your commit message `... I think all the permissions we wish to extend through a resigned URL have to be available to the AWS client`.  I'm not sure what you mean by that but if your goal is to make the contents of this bucket available from AWS CLI  then you may need an extra set of policies to make that happen.

Reply:
Our code will return temporary presigned URLs through the API to upload a file to this bucket, and to later download the file from this bucket. I had read that the AWS client that asks for these URLs cannot assign them more privileges than it itself possesses, but it appears our AWS S3 client has full S3 bucket access.

Reply:
We don't need this. This particular bucket policy grants access to Synapse. Since we don't want to expose this to Synapse, we don't want to include this bucket policy.

Reply:
(removed)